benchmarks/FF_LHAPDF_bench.py

```diff
@@ -0,0 +1,149 @@
+"""Benchmark FFs from LHAPDF"""
+
+from banana import register
+
+from eko import interpolation
+from ekomark.benchmark.runner import Runner
+
+register(__file__)
+
+base_operator = {"ev_op_iterations": 50, "backward_inversion": "exact"}
```

Comment:
are you sure 50 is needed? I'm slightly worried we might run into numerical problems ... we usually use 10, isn't that sufficient?

Reply:
changed to 10, it seems to be fine too